configuration/csv-to-docs.md

```diff
@@ -1,32 +1,276 @@
+```
+Generated json doc for the health center
+```
+{
+  "type": "health_center",
+  "parent": {
+    "type": "district_hospital",
+    "parent": "",
+    "is_name_generated": "false",
+    "name": "District1",
+    "external_id": "",
+    "notes": "",
+    "geolocation": "",
+    "reported_date": 1544031155715,
+    "_id": "e8f9739a-5d37-5b1e-be3c-a571b2c2409b"
+  },
+  "is_name_generated": "FALSE",
+  "name": "HC1",
+  "reported_date": 1544031155715,
+  "_id": "8606a91a-f454-56e3-a089-0b686af3c6b7"
+}
+```
+
+**csv/person.csv**
+
+```
+reference_id:excluded,parent:place WHERE reference_id=COL_VAL,name,phone,sex,role,reported_date,patient_id
+p_hc1,health_center_1,Bob Johnson 1,+16143291527,male,manager,1552494835669,60951
+p_hc2,health_center_1,Bob Johnson 2,+16143291528,male,manager,1552494835669,60951
+
+```
+Generated json doc for the person
+```
+{
+  "type": "person",
+  "parent": {
+    "type": "health_center",
+    "parent": {
+      "type": "district_hospital",
+      "parent": "",
+      "is_name_generated": "false",
+      "name": "District1",
+      "external_id": "",
+      "notes": "",
+      "geolocation": "",
+      "reported_date": 1544031155715,
+      "_id": "e8f9739a-5d37-5b1e-be3c-a571b2c2409b"
+    },
+    "is_name_generated": "FALSE",
+    "name": "HC1",
+    "reported_date": 1544031155715,
+    "_id": "8606a91a-f454-56e3-a089-0b686af3c6b7"
+  },
+  "name": "Bob Johnson 1",
+  "phone": "+16143291527",
+  "sex": "male",
+  "role": "manager",
+  "reported_date": "1552494835669",
+  "patient_id": "60951",
+  "_id": "65c52076-84c5-53a2-baca-88e6ec6e0875"
+}
+
+```
+
+**csv/users.csv**
+```
+username,password,roles,phone,contact:person WHERE reference_id=COL_VAL,place:GET _id OF place WHERE reference_id=COL_VAL
+ac1,Secret_1,district_admin:red1,+123456789,p_hc1,health_center_1
+ac2,Secret_1,district_admin:supervisor,+123456789,p_hc2,health_center_1
+ac3,Secret_1,district_admin,+123456789,p_hc3,health_center_1
+ac4,Secret_1,district_admin,+123456789,p_hc4,health_center_1
+
+```
+This will generate the `users.csv` file in the working directory which is used by the `create-users` action. The contact and place fields should be resolved to the actual UUIDs.
+
+```
+p_hc1"username","password","roles","contact","phone","place"
+"ac1","Secret_1","district_admin:red1","65c52076-84c5-53a2-baca-88e6ec6e0875","+123456789","8606a91a-f454-56e3-a089-0b686af3c6b7"
+"ac2","Secret_1","district_admin:supervisor","b7d0dbd5-beeb-52a8-8e4c-513d0baece8e","+123456789","8606a91a-f454-56e3-a089-0b686af3c6b7"
+```
+
+
+### Using CSV Files on Google Drive
+
+In your project home directory create a new json file named `csvs-on-google-drive.json`. 
+
+The keys should be the name of the CSV file stored locally, and for the value use the google id for the document. The name of the file must match what the file is generating a doc for. See the documentation [here](https://github.com/medic/medic-conf#csv-file-name) for naming files. 
```

Comment:
is this link still right?

Reply:
> and for the value use the google id for the document.

What is this?

Reply:
> The name of the file must match what the file is generating a doc for

Which file? Didn't we just say it needs to be called `csvs-on-google-drive.json`?

Reply:
This is all still correct, but not clear enough!

```suggestion
The keys must be the relative path and filename where the CSV will be stored locally — see the CSV [file documentation](https://github.com/medic/medic-conf#csv-file-name) for the notation. The value of each key must be the ID of the corresponding file in Google Drive — the ID can be obtained from the URL eg `https://docs.google.com/spreadsheets/d/{FILE_ID}/edit`. 
```